cmd/ops/flags_helpers.go

```diff
@@ -14,10 +14,15 @@ const (
 var flags = struct {
 	outputFormat string
 	schemaFile   string
```

Comment:
I image this is till here for `unused` that you said will go next. Just checking

Reply:
That's correct. Will be removed once `unused` supports multiple `--schema`.